toonz/sources/tnztools/brushtool.h

```diff
@@ -33,11 +33,14 @@ class BluredBrush;
 
 struct BrushData final : public TPersist {
   PERSIST_DECLARATION(BrushData)
-
+  // frameRange, snapSensitivity and snap are not included
+  // for some reason, including them causes presets to break
```

Comment:
Confirmed that it works fine. Then can you please remove this line?
As excluding snap and frame range is intentional for now.